com.icons8.Lunacy.yml

```
@@ -0,0 +1,70 @@
+app-id: com.icons8.Lunacy
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: lunacy
+build-options:
+  no-debuginfo: true
+tags:
+  - proprietary
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=cups
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=xdg-config/fontconfig:ro
+  - --talk-name=org.gtk.vfs.*
+  # Network needed for remote asset importing
+  - --share=network
+
+modules:
+  - name: lunacy
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/bin
+      - install -D lunacy ${FLATPAK_DEST}/bin/lunacy
+      - install -D /usr/bin/ar -t ${FLATPAK_DEST}/bin
+      - mkdir -p ${FLATPAK_DEST}/lib
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so ${FLATPAK_DEST}/lib
+      - install -D apply_extra ${FLATPAK_DEST}/bin/apply_extra
+      - install -D -t ${FLATPAK_DEST}/share/export/share/icons/scalable com.icons8.Lunacy.svg
```

```suggestion
      - install -D -t ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps com.icons8.Lunacy.svg
```

This is where icons are installed.